Examples/Swift/ClusteringExample.swift

```diff
@@ -33,39 +33,37 @@ class ClusteringExample_Swift: UIViewController, MGLMapViewDelegate {
 
         // Show unclustered features as icons. The `cluster` attribute is built into clustering-enabled source features.
         let ports = MGLSymbolStyleLayer(identifier: "ports", source: source)
-        ports.iconImageName = MGLStyleValue(rawValue: "icon")
-        ports.iconColor = MGLStyleValue(rawValue: UIColor.darkGray.withAlphaComponent(0.9))
-        ports.predicate = NSPredicate(format: "%K != YES", "cluster")
+        ports.iconImageName = NSExpression(forConstantValue: "icon")
+        ports.iconColor = NSExpression(forConstantValue: UIColor.darkGray.withAlphaComponent(0.9))
+        ports.predicate = NSPredicate(format: "cluster != YES")
         style.addLayer(ports)
 
         // Color clustered features based on clustered point counts.
         let stops = [
-            20:  MGLStyleValue(rawValue: UIColor.lightGray),
-            50:  MGLStyleValue(rawValue: UIColor.orange),
-            100: MGLStyleValue(rawValue: UIColor.red),
-            200: MGLStyleValue(rawValue: UIColor.purple)
+            20:  UIColor.lightGray,
+            50:  UIColor.orange,
+            100: UIColor.red,
+            200: UIColor.purple
         ]
 
         // Show clustered features as circles. The `point_count` attribute is built into clustering-enabled source features.
         let circlesLayer = MGLCircleStyleLayer(identifier: "clusteredPorts", source: source)
-        circlesLayer.circleRadius = MGLStyleValue(rawValue: NSNumber(value: Double(icon.size.width) / 2))
-        circlesLayer.circleOpacity = MGLStyleValue(rawValue: 0.75)
-        circlesLayer.circleStrokeColor = MGLStyleValue(rawValue: UIColor.white.withAlphaComponent(0.75))
-        circlesLayer.circleStrokeWidth = MGLStyleValue(rawValue: 2)
-        circlesLayer.circleColor = MGLSourceStyleFunction(interpolationMode: .interval,
-                                                          stops: stops,
-                                                          attributeName: "point_count",
-                                                          options: nil)
-        circlesLayer.predicate = NSPredicate(format: "%K == YES", "cluster")
+        circlesLayer.circleRadius = NSExpression(forConstantValue: NSNumber(value: Double(icon.size.width) / 2))
```

Comment:
If it isn’t too confusing, `(Double(icon.size.width) / 2) as NSNumber` is another way of writing this.